gui/gm-editor.lua

```diff
@@ -577,6 +615,14 @@ end
 
 local function get_editor(args)
     if #args~=0 then
+        if args[1]=='dev' or args[1]=='edit' then
+            read_only = false
+            table.remove(args,1)
+        end
+        if args[1]=='safe' or args[1]=='read' then
+            read_only = true
+            table.remove(args,1)
+        end
```

Comment:
For consistency with other tools, I would prefer to use `utils.processArgsGetopt()` or another option here if possible. That would allow things like
```
gui/gm-editor -r
gui/gm-editor --read-only
```
or, I suppose
```
gui/gm-editor -e
gui/gm-editor --editable
```

Reply:
I thought about rewriting the arg processing, but didn't want to touch the existing functionality.  The `dialog` and (undocumented?) `free` args would need to be changed too, and I'm not sure how the getopt code would deal with passing an expression to be evaluated.

Could a getopt rewrite be done in a separate PR?

Reply:
I didn't realize there were existing options. Disregard.